style.css

```diff
@@ -0,0 +1,712 @@
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+  padding: 15px 38px 15px 16px;
+}
+
+form button {
+  width: 140px;
+  height: 48px;
+  color: #6070ff;
+  background: #fff;
+  border: none;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 8px;
+}
+
+form button:hover {
+  background: #661d79;
+  box-shadow: 1px 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  transition: all 0.5s ease-out;
+}
+
+.contact-hr {
+  background: #0d151e;
+  border-radius: 10px;
+  width: 134px;
+  height: 5px;
+}
+
+.sec-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.top-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup-image {
+  max-width: 100%;
+}
+
+.button-image {
+  width: 20px;
+}
+
+@media (min-width: 768px) {
+  body {
+    padding: 24px;
+  }
+
+  .top {
+    border-bottom-left-radius: 100px;
+  }
+
+  .header {
+    padding: 14px 80px;
+  }
+
+  .humburger {
+    display: none;
+  }
+
+  .nav-menue2 {
+    display: flex;
+    flex-direction: row;
+    color: #396df2;
+    gap: 15px;
+  }
+
+  .section {
+    background-position: top center;
+    background-size: 90% 90vh;
+    justify-content: center;
+    margin-top: 50px;
+    margin-bottom: 76px;
+    width: 100%;
+    border-bottom-left-radius: 50px;
+  }
+
+  .description p {
+    height: 6rem;
+    overflow: hidden;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .title {
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .info {
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .works {
+    padding: 80px;
+    gap: 142px;
+  }
+
+  #card1 {
+    display: flex;
+    flex-direction: row;
+    gap: 46px;
+  }
+
+  #card2 {
+    padding: 24px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 46px;
+  }
+
+  #card3 {
+    border-radius: 24px;
+    padding: 24px;
+    display: flex;
+    flex-direction: row;
+    gap: 46px;
+  }
+
+  #card4 {
+    border-radius: 24px;
+    padding: 24px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 46px;
+  }
+
+  .description {
+    width: 100%;
+    margin-top: 10px;
+  }
+
+  .img-container {
+    height: 350px;
+    width: 500px;
+  }
+
+  .button-container {
+    padding: 12px;
+    width: 150px;
+    height: 48px;
+  }
+
+  .button-container3 {
+    background: #4053fc;
+    box-shadow: 1px 8px 16px rgba(64, 83, 252, 0.24);
+    color: #fff;
+    padding: 12px;
+    width: 124px;
+    height: 48px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    border-top-right-radius: 100px;
+    width: 96%;
+  }
+
+  .container2 {
+    display: flex;
+    flex-direction: row;
+    border-top-right-radius: 100px;
+  }
```

Comment:
- [ ] Please ensure that this container children are aligned properly according to the given template. 👍 

## Figma Design
![langstack](https://user-images.githubusercontent.com/103386183/206170400-6f327a56-bb99-418d-a8fc-46958cc3e07a.png)